packages/palette/src/helpers/injectGlobalStyles.tsx

```diff
@@ -36,9 +38,9 @@ export function injectGlobalStyles<P>(
     }
 
     html, body {
-      font-family: 'AGaramondPro-Regular';
-      font-size: 16px;
-      line-height: 24px;
+      font-family: ${themeProps.fontFamily.sans.regular as string};
+      font-size: ${themeProps.fontSizes.size3};
```

Comment:
This should be `size5` => `16px`?

Changing this will change the value of `1rem`

Reply:
We probably want to like alias these as ROOT_FONT_SIZE, ROOT_LINE_HEIGHT, etc, in that tokens file just so it's clear if any changes are made in that file.

Reply:
Ah, yeah, that's a good point... should add it to the themeProps instead though? That tokens file is a bit buried.

Reply:
https://github.com/artsy/palette/blob/master/packages/palette/src/elements/Text/tokens.ts is pretty clear still. My hope is that once we retire `Sans/Serif` we can clean up the theme a bit.

Reply:
Oh, I see what you're saying — yah, makes sense (I only carved out the Text/tokens file because of the confusion with the older tokens.)

Reply:
Need to start adding some deprecation notices soon.